core/src/Threads/Kokkos_Threads_Instance.hpp

```diff
@@ -403,7 +403,7 @@ class ThreadsInternal {
   static void start(void (*)(ThreadsInternal &, const void *), const void *);
 
 #ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
-  KOKKOS_DEPRECATED static int in_parallel();
+  static int in_parallel();
 #endif
```

Comment:
???
not captured in the description

Reply:
It's internal in the `Impl` namespace so we don't need to deprecate (which causes warnings when used in the non-`Impl` overload).

Reply:
Updated the description of the pull request.